.github/workflows/libjs-test262.yml

```diff
@@ -133,3 +133,39 @@ jobs:
         continue-on-error: true
         working-directory: libjs-test262
         run: ./per_file_result_diff.py -o per-file-bytecode-master.json -n ../libjs-website/test262/data/per-file-bytecode-master.json
+
+      - name: Build js
+        working-directory: libjs-test262/Build
+        run: |
+          ninja js
+
+      - name: Create js package
+        working-directory: libjs-test262/Build
+        run: |
+          mkdir -p pkg/bin pkg/lib
+          cp --preserve _deps/lagom-build/js pkg/bin
+
+          for lib in $(find _deps/lagom-build -type f -name liblagom-*.so.0.0.0) ; do
+            dst=$(basename -s ".0.0" ${lib})
+            cp --preserve ${lib} pkg/lib/${dst}
+          done
+
+          cd pkg
+
+          echo "#!/bin/sh" > js
+          echo "DIRNAME=\"\$(dirname \$(readlink -f \$0))\"" >> js
+          echo "export LD_LIBRARY_PATH=\"\${DIRNAME}/lib\"" >> js
+          echo "exec \"\${DIRNAME}/bin/js\" \"\$@\"" >> js
```

Comment:
How about a heredoc?

Reply:
Done, for some reason I assumed that'd be a pain with indented yaml, but it just works :tm: